client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -104,13 +104,13 @@ export class Checkout extends React.Component {
 		window.scrollTo( 0, 0 );
 	}
 
-	componentWillReceiveProps( nextProps ) {
+	UNSAFE_componentWillReceiveProps( nextProps ) {
```

Comment:
_This is a general observation, I'm not saying whether it does or does not hold here._

Often, these `UNSAFE_componentWill*` lifecycle methods can be swapped for a `componentDid*` version that roughly corresponds:

`UNSAFE_componentWillMount` ➡️  `componentDidMount`
`UNSAFE_componentWillReceiveProps( nextProps )` ➡️  `componentDidUpdate( prevProps )`